code/game/objects/structures/dropship_ammo.dm

```diff
@@ -28,6 +28,7 @@
 	var/point_cost = 0 //how many points it costs to build this with the fabricator, set to 0 if unbuildable.
 	///Type of ammo
 	var/ammo_type
+	var/Penetrate_Cave = FALSE
```

Comment:
1) autodoc (see the pins in dev gen)
2) penetrate_cave, not Penetrate_Cave
3) This could probably be in a flag with transferable_ammo, rather than a bool